tests/fixture/tmpnet/network.go

```diff
@@ -793,11 +769,11 @@ func (n *Network) CreateSubnets(ctx context.Context, log logging.Logger, apiURI
 			return err
 		}
 
-		// Persist the chain configuration
-		if err := subnet.Write(n.GetSubnetDir(), n.GetChainConfigDir()); err != nil {
+		// Persist the subnet
```

Comment:
nit maybe a tiny bit clearer:

```suggestion
		// Persist the subnet tmpnet config
```

Persisting/writing subnet is a bit odd I think 🤔 Maybe I'm being too nitpicky as well!